shared/middleware/helpers/auth_helpers.rb

```diff
@@ -18,6 +18,14 @@ def current_user
   @dashboard_user ||= DASHBOARD_DB[:users][id: current_user_id]
 end
 
+# Returns true if the current user is under 13 or if age is unknown.
+def under_13?
+  return true unless current_user
+  birthday = current_user[:birthday]
+  age = ((Date.today - birthday) / 365).to_i
+  age < 13
+end
```

Comment:
Not sure there is a great way, thus my asking questions. Perhaps in [user_helpers.rb](https://github.com/code-dot-org/code-dot-org/blob/staging/lib/cdo/user_helpers.rb)?